spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -245,6 +245,7 @@ protected CloseableHttpClient newClient() {
 			httpClientBuilder.setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE);
 		}
 		return httpClientBuilder.setConnectionManager(newConnectionManager())
+				.disableContentCompression()
```

Comment:
are you referring to disabling the content compression? it appears to be in both the original pull request and the 2.0 branch
original fix: https://github.com/joshiste/spring-cloud-netflix/blob/d9b50148c1637a5feb38becf2aa66e6694a8ba8e/spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java#L261
2.0 branch: https://github.com/spring-cloud/spring-cloud-netflix/blob/2.0.x/spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java#L261